client/utils.go

```diff
@@ -50,6 +50,12 @@ func ReadPageRequest(flagSet *pflag.FlagSet) *query.PageRequest {
 	limit, _ := flagSet.GetUint64(flags.FlagLimit)
 	countTotal, _ := flagSet.GetBool(flags.FlagCountTotal)
 
+	// FlagPage is deprecated, provided for backwards compatibility
+	page, _ := flagSet.GetUint64(flags.FlagPage)
+	if page > 0 && offset == 0 {
+		offset = page * limit
```

Comment:
Actually, it seems to me that the first page is not 0 but 1

https://github.com/cosmos/cosmos-sdk/blob/32de79d0aa0062bd1b0aa038601fa76abbd7b024/x/gov/client/utils/query.go#L13

which means:

```suggestion
		offset = (page - 1) * limit
```

Reply:
Yeah, default page is `1`, thanks @amaurymartiny